web/public/404.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html><head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
```

Comment:
All contents of `web/public` is generated via `hugo` utility. Since github build environment don't have it available, we need to generate the content and merge it into git. That will work for now.